api/api.go

```diff
@@ -186,6 +186,18 @@ func HandleJobRequest(cache job.JobCache, db job.JobDB) func(w http.ResponseWrit
 	}
 }
 
+// HandleDeleteAllJobs is the handler for deleting all jobs
+// DELETE /api/v1/job/all
+func HandleDeleteAllJobs(cache job.JobCache, db job.JobDB) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if err := job.DeleteAll(cache, db); err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
@ajvb What should be done with errors in situations like this? Should be fine to send to the user as JSON, yes? E.g., `{"error": "..."}`

Reply:
Yeah absolutely, will want to use something similar to `http.Error` - https://github.com/ajvb/kala/pull/160/files#diff-604f4192deea5adf9c159e5f88f64630R130 - as that is what we are currently using.

I'm going to make an issue for the fact that we should actually send back json

Reply:
I did it like it was done for simple deletion endpoint at https://github.com/ajvb/kala/pull/160/files#diff-604f4192deea5adf9c159e5f88f64630R179
But I can fix it after merging #165.

Reply:
:+1: Makes sense. #165 is now merged in

Reply:
Done!